services/web/server/tests/unit/with_dbs/02/test_projects_groups_handlers.py

```diff
@@ -251,3 +251,44 @@ async def test_projects_groups_full_workflow(
                 "delete": True,
             },
         }
+
+
+@pytest.mark.parametrize("user_role", [UserRole.USER])
```

Comment:
do you check somewhere that it is not working for ANONYMOUS or so?

Reply:
did a new test to check that.